### PR TITLE
Ensure correct `graphicId` HTML element / labels

### LINKED ISSUE #4473: Ensure correct `graphicId` HTML element / labels
See https://github.com/mozilla/fxa/pull/4465#discussion_r389880425

Places in which we set a graphic through a class on a `div` with a `background-image` need some `a11y` love.

These should preferably be `img` tags for screen readers, however, our image file names receive a hash via grunt on stage/prod build so a simple path to the icon will break in stage. A `data:image/svg+xml;utf8,<svg...` src could be the way to go, however, this may be tricky since we use `getGraphicsId` to check for which class name to set on the div.

At the very least, we can instead place `role="img"` and `aria-label="alt text"` on the divs which will read as images. Reference the link above to see how this was done in one file. Closing this ticket should explore if we can use `img` but if not, proper roles/aria-labels on all divs that receive a `graphicId` will suffice.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-1328)
┆Issue Number: FXA-1328

### PR DESCRIPTION
**Pull request by imskr**
added `role` and `aria-label` 

closes #4473

@jodyheavener

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {{/isSignedIn}}
   {{/showSuccessMessage}}
   <section>
-    <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+    <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Successfully authenticated{{/t}}">{{#t}}Success{{/t}}</div>
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache]: I think we can remove the text content of these elements (e.g. `{{#t}}Success{{/t}}`). This was likely an older method of aiding screen readers with visibly hidden text, but now that we're moving to a better approach we can probably remove them.

**Comment by imskr** [comment]
@jodyheavener Thanks for your input. I've made the changes as you suggested.

**Comment by imskr** [comment]
okay, I'll be pushing the changes now.

**Comment by imskr** [comment]
@jodyheavener Please review now! Thanks.

File: packages/fxa-content-server/app/scripts/templates/pair/auth_complete.mustache [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}">{{#t}}Success{{/t}}</div>
+    <div class="graphic {{ graphicId }}" role="img" aria-label="{{#t}}Success{{/t}}"></div>
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/templates/pair/auth_complete.mustache]: `{{#t}}Successfully connected{{/t}}`

File: packages/fxa-content-server/app/scripts/templates/pair/index.mustache [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}"></div>
+    <div class="graphic {{ graphicId }}" role="img" aria-label="Connect another device"></div>
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/templates/pair/index.mustache]: `{{#t}}Scan to connect another device{{/t}}` (also, note the addition of the translation tag)

File: packages/fxa-content-server/app/scripts/templates/pair/success.mustache [hunk 4]
```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}">{{#t}}Device connected{{/t}}</div>
+    <div class="graphic {{ graphicId }}" role="img" aria-label="{{#t}}Device connected{{/t}}"></div>
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/templates/pair/success.mustache]: `{{#t}}Device successfully paired{{/t}}`

File: packages/fxa-content-server/app/scripts/templates/sms_sent.mustache [hunk 5]
```diff
@@ -6,7 +6,7 @@
     <div class="success"></div>
     <div class="error"></div>
 
-    <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+    <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Success{{/t}}"></div>
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/templates/sms_sent.mustache]: `{{#t}}We've sent you a text message{{/t}}`

File: packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache [hunk 6]
```diff
@@ -14,7 +14,7 @@
     {{/isSignedIn}}
   {{/showSuccessMessage}}
   <section>
-    <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+    <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Successfully authenticated{{/t}}"></div>
```
[the remark this answers is not in the document]
Comment (imskr) [reply packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache]: It is good. I will make the changes

**Comment by imskr** [comment]
@jodyheavener Done

**Comment by imskr** [comment]
@jodyheavener yes, I will do it.

**Comment by imskr** [comment]
It's updated now @jodyheavener

**Comment by jodyheavener** [comment]
@imskr Thank you for taking the time to contribute 🙂

**Pull request merged**